src/QirRuntime/test/QIR-static/qir-test-qubits-results.qs

```diff
@@ -1,3 +1,4 @@
+//

```

Comment:
Clarification: This change enables the (C/C++) syntax highlighting in my editor for this .qs file.
Let me know if you would like me to remove this change.

Reply:
Would you mind adding the licensing boilerplate instead (all our source files are required to have it, but apparently I've missed this one)?

```
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License.
```

Reply:
As a slight nit, the most up to date version of the header does not have "All rights reserved." It's not essential to go fix in existing headers, but is recommended for all new headers.

Reply:
Fixed.